hls4ml/converters/pytorch_to_hls.py

```diff
@@ -98,7 +88,7 @@ def decorator(function):
     'elu': 'ELU',
     'prelu': 'PReLU',
     'sigmoid': 'Sigmoid',
-    'layer_threshold': 'Threshold',
+    '_threshold': 'Threshold',
```

Comment:
I just wanted confirmation that this is supposed to have the leading underscore.

Reply:
Yes. Once we get rid of prefixing with `layer` this one indeed remains as `_threshold`. It corresponds to the [torch.functional call](https://github.com/pytorch/pytorch/blob/main/torch/nn/functional.py#L1431). I don't understand the reasoning behind the naming :smiley: